vMenu/menus/MpPedCustomization.cs

```diff
@@ -1831,197 +1831,205 @@ private async Task SpawnSavedPed(bool restoreWeapons)
             {
                 return;
             }
-            if (IsModelInCdimage(currentCharacter.ModelHash))
+            int theplayerped = PlayerPedId();
```

Comment:
You can use `Game.PlayerPed.Handle` instead, should be more efficient. But to make it even better, I'm pretty sure you can check (almost) all of these things below by just using the `Game.PlayerPed` object's functions/properties.